tests/framework/installer/ceph_installer.go

```diff
@@ -964,6 +804,7 @@ func (h *CephInstaller) addCleanupPolicy(namespace string) error {
 		return fmt.Errorf("failed to get ceph cluster. %+v", err)
 	}
 	cluster.Spec.CleanupPolicy.DeleteDataDirOnHosts = "yes-really-destroy-data"
+	cluster.Spec.CleanupPolicy.SanitizeDisk.SanitizeDiskConfirmation = "yes-really-sanitize-disks"
```

Comment:
Are you relying on all the defaults for wiping the disks? How about setting the test cleanup policy in ceph_manifests.go?

Reply:
Yes, the defaults are enough.